universal-login-relayer/test/unit/http/middlewares/errorHandler.ts

```diff
@@ -13,14 +13,12 @@ describe('UNIT: Error Handler', () => {
 
     it('specific exceptions', () => {
       expect(getCodeForException(new InvalidSignature())).to.eq(400);
-      expect(getCodeForException(new InvalidContract('0x0'))).to.eq(400);
       expect(getCodeForException(new InvalidTransaction('0x0'))).to.eq(400);
       expect(getCodeForException(new NotEnoughSignatures(3, 2))).to.eq(400);
       expect(getCodeForException(new InvalidMessage('0x0'))).to.eq(404);
       expect(getCodeForException(new InvalidENSDomain('domain'))).to.eq(404);
       expect(getCodeForException(new NotEnoughGas())).to.eq(402);
       expect(getCodeForException(new NotEnoughBalance())).to.eq(402);
-      expect(getCodeForException(new NotEnoughTokens())).to.eq(402);
```

Comment:
Why these tests are removed?

Reply:
I didn't be sure about adding ValidationFailed from commons to getCodeForException. But i can handle these two errors.